app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -82,7 +81,7 @@ class ProfileFragment : Fragment() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem?): Boolean {
-        when (item?.getItemId()) {
+        when (item?.itemId) {
```

Comment:
I thought this file was part of change, since this file is not a part of changes, please revert this